src/App.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700&family=Roboto+Slab:wght@100;200;300&display=swap');
+
 .App {
   text-align: center;
 }
```

Comment:
- [ ] Let's align all the elements on the page properly so as to match the design guideline. The logo `Bookstore CMS` and the `user` avatar should be aligned with the cards and the from should align with cards too. This can be achieved by giving them the same `width` as the cards. I have highlighted the suggested changes below;


Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/9s1X90S.png) | ![](https://i.imgur.com/YHq7J1J.png)